`Uncaught Error: Cannot find module './common/helpers'` on 0.8.14

Looks like the helpers added in #603 were not added to the package so trying to import it fails:
```bash
npm install solc@0.8.14
node_modules/solc/solc.js --version
```
```
Error: Cannot find module './common/helpers'
```
Also:
```bash
node -e 'require("solc")'
```
```
Error: Cannot find module './common/helpers'
```

Also, we need to investigate why this didn't fail any tests. We specifically build the package in CI and install it and that test should have caught it.

Sure. Actually looks like @ekpyron has just created a fix (#623) so please just take a look if nothing is missing there.

And I'm investigating why tests did not catch this. I just reran one of them locally and it did catch it. For reference, I ran this (equivalent to the `hardhat-sample-project` job in CI):

```bash
git clone https://github.com/ethereum/solc-js
pushd solc-js
npm install
npm run updateBinary
npm run build
BYPASS_SAFETY_CHECK=true npm pack ./dist
mv solc-*.tgz ../solc-js.tgz
popd

git clone --depth 1 "https://github.com/nomiclabs/hardhat-hackathon-boilerplate" boilerplate/
rm boilerplate/package-lock.json
pushd boilerplate/
npm update hardhat
pushd node_modules/hardhat/
npm install ../../../solc-js.tgz
popd
popd

find boilerplate/ \
    -path "*/solc/wrapper.js" \
    -not -path "boilerplate/node_modules/hardhat/node_modules/solc/wrapper.js" \
    -printf "%h\n" | xargs --verbose rm -r

solc_version=$(jq --raw-output .version solc-js/package.json)

pushd boilerplate/
sed -i 's|pragma solidity [^;]\+;|pragma solidity *;|g' contracts/Token.sol
{
    echo "  const {TASK_COMPILE_SOLIDITY_GET_SOLC_BUILD} = require('hardhat/builtin-tasks/task-names');"
    echo "  const assert = require('assert');"
    echo
    echo "  subtask(TASK_COMPILE_SOLIDITY_GET_SOLC_BUILD, async (args, hre, runSuper) => {"
    echo "      assert(args.solcVersion == '${solc_version}', 'Unexpected solc version: ' + args.solcVersion);"
    echo "      return {"
    echo "          compilerPath: '$(realpath "../solc-js/soljson.js")',"
    echo "          isSolcJs: true,"
    echo "          version: args.solcVersion,"
    echo "          longVersion: args.solcVersion"
    echo "      };"
    echo "  })"
    echo "  module.exports = {solidity: '${solc_version}'};"
} >> hardhat.config.js
npm run test
popd
```
```
...
> hardhat-hackathon-boilerplate@1.0.0 test
> hardhat test

An unexpected error occurred:

Error: Cannot find module './common/helpers'
Require stack:
- /tmp/test/boilerplate/node_modules/hardhat/node_modules/solc/linker.js
...
```
So they really should have cought it.

--- a/package.json
+++ b/package.json
@@ -43,7 +43,8 @@
     "solc.js",
     "soljson.js",
     "translate.js",
-    "wrapper.js"
+    "wrapper.js",
+    "common/helpers.js"
   ],
   "author": "chriseth",
   "license": "MIT",

